manifests/dbmodules.pp

```diff
@@ -16,15 +16,15 @@
 #   value should be db2 for the DB2 module and kldap for the LDAP module.
 #
 # [*disable_last_success*]
-#   If set to true, suppresses KDC updates to the “Last successful
-#   authentication” field of principal entries requiring preauthentication.
+#   If set to true, suppresses KDC updates to the â€œLast successful
+#   authenticationâ€� field of principal entries requiring preauthentication.
```

Comment:
" character is shown incorrectly

Reply:
I guess this is just `U+2019` not displayed properly . Nothing wrong on my side and this shouldn't bother Puppet.